rest/api_test.go

```diff
@@ -3137,6 +3137,37 @@ func TestDocSyncFunctionExpiry(t *testing.T) {
 	log.Printf("value: %v", value)
 }
 
+func TestCustomCookieName(t *testing.T) {
+
+	var rt RestTester
+	defer rt.Close()
+
+	customCookieName := "TestCustomCookieName"
+	rt.DatabaseConfig = &DbConfig{
+		Name: "db",
+		SessionCookieName: base.StringPointer(customCookieName),
```

Comment:
Since you're not passing a literal here, I think base.StringPointer just obfuscates the code (compared to &customCookieName).

Reply:
Fixed